frontend/src/app/pages/exercises/exercise/shared/emergency-operations-center/send-alarm-group-interface/send-alarm-group-interface.component.ts

```diff
@@ -98,6 +103,13 @@ export class SendAlarmGroupInterfaceComponent implements OnDestroy {
                     color: 'success',
                 });
             }
+            this.apiService.proposeAction({
+                type: '[Emergency Operation Center] Add Log Entry',
+                message: `Alarmgruppe ${alarmGroup.name} wurde alarmiert zu ${targetTransferPoint.externalName}!`,
```

Comment:
TODO: I don't like the sound of this message yet...
Maybe: 

```ts
`${alarmGroup.name} wurde nach ${targetTransferPoint.externalName} alarmiert!`
```

Reply:
@christianzoellner

Reply:
We will merge this soon - I would create another PR for changing this message.